horizons/world/player.py

```diff
@@ -38,7 +38,16 @@
 
 
 class Player(ComponentHolder, WorldObject):
-	"""Class representing a player"""
+	"""Class representing a player
+
+	@type session: horizons.session.Session
+	@param session: Session instance
+	@param worldid: player's worldid
+	@param name: user-chosen name
+	@param color: color of player (as Color)
+	@param clientid: id of client
+	@param inventory: {res: value} that are put in the players inventory
+	"""
```

Comment:
Well, that only checks whether there is any docstring at all in `__init__`. Obviously we have more docstrings in classes.

Oh well, I'm kinda getting lost with the goal of this PR here honestly.